### PR TITLE
CVL-16: Add home page tiles for various top-level workflows

### PR DESCRIPTION
**Pull request by timharrison-moj**
Adds ability for a tiled home landing page.
Example top-level workflows added.

### CODE REVIEW

File: assets/sass/components/_card.scss [hunk 1]
```diff
@@ -0,0 +1,100 @@
+/* ==========================================================================
```
Comment (yasinmustafa) [inline assets/sass/components/_card.scss:1]: Is this 'card' component a pattern they use in other services?

File: assets/js/card.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+window.onload = function () {
```
Comment (yasinmustafa) [inline assets/js/card.js:1]: presumably they use this card component elsewhere and  it requires this jquery?

**Review by yasinmustafa — APPROVED** [review]
LGTM - A couple of questions

File: assets/sass/components/_card.scss (hunk 1 above)
Comment (timharrison-moj) [reply assets/sass/components/_card.scss:1]: Yes - I stole it from book a video link, but its in several others.

File: assets/js/card.js (hunk 2 above)
Comment (timharrison-moj) [reply assets/js/card.js:1]: yes. Its a way of making the whole tile clickable, and for the click to follow the link defined in the card header.

Comment (timharrison-moj) [reply assets/js/card.js:1]: Actually, just checking that, it doesn't. Either it doesn't do that or the script is not active. I'll dig a bit more in next PR.

**Pull request merged**